README.md

```diff
@@ -1,2 +1,30 @@
-# cumulus-library-suicidality-icd10
-Validation of ICD-10 code usage of Suicidality Cases Compared to Patient Chart Review
+# How reliable are ICD-10 diagnosis codes to identify cases of pediatric suicidality?
+
+This study repository includes:
+
+* **Suicidality ICD10 case definition** (SQL on FHIR) 
+* [Propensity Score Matching](https://github.com/smart-on-fhir/cumulus-library/blob/main/tests/test_psm.py) (PSM) to select "case matched comparators"
```

Comment:
This probably shouldn't link to the unit test. Some other options:
- link to [the documentation](https://docs.smarthealthit.org/cumulus/library/statistics/propensity-score-matching.html), and note that the module was generated based on the work done for this study
- link to the [PSM module](https://github.com/smart-on-fhir/cumulus-library/blob/main/cumulus_library/builders/psm_builder.py), with the same comment above
- The supplement document